releasenotes/notes/rename-tag-command.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+releaseNotes:
+  - |
+    **Added** `istioctl x tag` command group to `istioctl x revision tag` subcommand
```

Comment:
It looks like this was a _move_ of the `tag` command. But, I also notice looking at the istio.io reference that the `tag` command isn't listed, so maybe this is a new command for 1.10. If the later, than maybe it just needs to say that the subcommand was added.

Reply:
Ah good call. I'll edit the existing release note instead of adding a new one, thanks